File: baekjoon/팩토리얼_2.js
const input = +require("fs").readFileSync("/dev/stdin").toString().trim();

if (input === 0) {
  console.log(1);
  return;
}

let factorial = 1;

for (let i = 2; i <= input; i++) {
  factorial *= i;
}

console.log(factorial);

File: baekjoon/팩토리얼_2.rs
fn main() {
    let mut st = String::new();
    std::io::stdin().read_line(&mut st).unwrap();
    let num = st.trim().parse::<u64>().unwrap();

    let mut factorial: u64 = 1;    
    for n in 2..=num {
        factorial *= n;
    }

    println!("{}", factorial);
}